contracts/estate/EstateRegistry.sol

```diff
@@ -0,0 +1,236 @@
+pragma solidity ^0.4.22;
+
+import "zeppelin-solidity/contracts/math/SafeMath.sol";
+import "zeppelin-solidity/contracts/token/ERC721/ERC721Token.sol";
+import "zeppelin-solidity/contracts/ownership/Ownable.sol";
+
+import "../metadata/MetadataHolderBase.sol";
+
+
+contract PingableDAR {
+  function ping() public;
+  function ownerOf(uint256 tokenId) public returns (address);
+  function safeTransferFrom(address, address, uint256) public;
+}
+
+
+// @nico TODO: Standalone Storage.
+// @nico TODO: Document functions.
+contract EstateRegistry is ERC721Token, Ownable, MetadataHolderBase {
+  using SafeMath for uint256;
+
+  PingableDAR public dar;
+
+  // From Estate to list of owned tokenIds (LANDs)
+  mapping(uint256 => uint256[]) public estateTokenIds;
+
+  // From tokenId (LAND) to its owner estateId of owned tokenIds
+  mapping(uint256 => uint256) public tokenIdEstate;
+
+  // From Estate to mapping of tokenId to index on the array above
+  mapping(uint256 => mapping(uint256 => uint256)) public estateTokenIndex;
+
+  // Metadata of the Estate
+  mapping(uint256 => string) internal estateData;
+
+  // Operators of the Estate
+  mapping (uint256 => address) public updateOperator;
```

Comment:
I think we already have this storage as part of the [ERC721BasicToken](https://github.com/OpenZeppelin/openzeppelin-solidity/blob/master/contracts/token/ERC721/ERC721BasicToken.sol#L27)

Reply:
I think it functions in a different way. You can set a token approval via `approve` and the description says:

> @dev Approves another address to transfer the given token I

We just want the updateOperator to be able to manage the metadata (for now at least)